tests/fast-integration/eventing-test/utils.js

```diff
@@ -41,9 +41,9 @@ const kymaVersion = process.env.KYMA_VERSION || '';
 const kymaStreamName = 'sap';
 const isSKR = process.env.KYMA_TYPE === 'SKR';
 const skrInstanceId = process.env.INSTANCE_ID || '';
-const testCompassFlow = process.env.TEST_COMPASS_FLOW || false;
-const isUpgradeJob = process.env.EVENTING_UPGRADE_JOB || false;
-const isUpgradeJob2ndReconcile = process.env.EVENTING_UPGRADE_2ND_RECONCILE_JOB || false;
+const testCompassFlow = process.env.TEST_COMPASS_FLOW === 'true';
```

Comment:
Why did you change true from bool to string?

Reply:
The ENV is a string. What happening was the if the ENV is defined then the `testCompassFlow` is true, regardless of whatever is in the value if this ENV.

Reply:
Okay, I see. Thank you!